src/components/TalentPage/TalentPageLayout.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { FC, FormEvent, Suspense, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Controller, useForm } from 'react-hook-form';
+import { useMemo } from 'react';
+import {
+	Wrapper,
+	Title,
+	Input,
+	BorderStyled,
+	MainBlockWrapper,
+	Label,
+	Button,
+	ButtonBlock,
+	SkillsButtonsBlock,
+	WrapperSidePanel,
+	InputContainer,
+	IconSearch,
+	ProfileBlock,
+	PaginationBlock,
+} from './TalentPageLayout.style';
+import { ImSearch } from 'react-icons/im';
+import { ISkill } from 'components/jobPost/interfaces';
+import { skillsMock } from 'components/jobPost/dataChanges';
+import { Filter, ICategoryBE, SearchSubmitForm } from './interfaces';
+import Select from 'react-select';
+import { selection } from 'components/jobPost/dataChanges';
+import { BsArrowLeftCircle } from 'react-icons/bs';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import MyHiresCompany from './myhires/MyHiresCompany';
+import MySavedTalent from './mysaved/MySavedTalent';
+import FilterProfileUser from './FilterProfileUser';
+import { useGetFilterProfileQuery } from 'service/httpService';
+import Pagination from './Pagination';
+const discover = 'discover';
+const hires = 'hires';
+const saved = 'save';
+
+const TalentPageLayout: FC = () => {
+	const { t } = useTranslation();
+	const [skillsOption, setSkillsOptions] = useState<ISkill[]>(skillsMock);
+	const [select, setSelect] = useState<ICategoryBE>();
+	const [search, setSearch] = useState<string>('');
+	const [showFilterList, setShowFilterList] = useState<boolean>(true);
+	const [active, setActive] = useState<{ [name: string]: string }>({ ['discover']: 'discover' });
+	const [currentPage, setCurrentPage] = useState<number>(1);
+	const paginate = (pageNumber: React.SetStateAction<number>) => setCurrentPage(pageNumber);
+	const {
+		control,
+		formState: { errors },
+	} = useForm<SearchSubmitForm>();
+
+	function useMediaQuery(query: string, defaultMatches = window.matchMedia(query).matches) {
+		const [matches, setMatches] = useState(window.matchMedia(defaultMatches.toString()).matches);
+
+		useEffect(() => {
+			const media = window.matchMedia(query);
+
+			if (media.matches !== matches) setMatches(media.matches);
+
+			const listener = () => setMatches(media.matches);
+
+			media.addEventListener('change', listener);
```

Comment:
hmm I dont realy see the reason why we need it?

Reply:
I used it for responsive design but now it deleted and I use useMediaQuery from usehooks-ts